plan/validator_test.go

```diff
@@ -71,6 +71,7 @@ func (s *testValidatorSuite) TestValidator(c *C) {
 			errors.New("[schema:1068]Multiple primary key defined")},
 		{"create table t(c1 int not null, c2 int not null, primary key(c1), primary key(c2))", true,
 			errors.New("[schema:1068]Multiple primary key defined")},
+		{"alter table t auto_increment=1", true, errors.New("[autoid:3]No support for setting auto_increment using alter_table")},
```

Comment:
I have added two tests. I think validator should return a `nil` in the validation of `alter table t add column c int auto_increment key`, because there are existed guard to avoid user commit something like `alter table t add column c int auto_increment key` or `alter table t modify c int auto_increment key` in `ddl` module.